src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/MalformedRoundaboutCheck.java

```diff
@@ -255,4 +268,70 @@ private static Double getCrossProduct(final Edge edge1, final Edge edge2)
         // Directly related to the direction of rotation/traffic
         return (vector1X * vector2Y) - (vector1Y * vector2X);
     }
+
+    /**
+     * Checks if an {@link Edge} has values that indicate it is crossable by another edge.
+     *
+     * @param edge
+     *            {@link Edge} to be checked
+     * @return true if it is a bridge or tunnel, or has a layer tag
+     */
+    private boolean ignoreCrossings(final Edge edge)
```

Comment:
Initially I thought this function was referring to Pedestrian crossings. Can you clarify the functions name that this looks at level considerations? ```ignoreBridgeTunnelCrossings``` ?